token-holders/how-to-vote.md

```diff
@@ -0,0 +1,29 @@
+{% hint style="info" %}
+Pyth governance is binding and can execute arbitrary code on-chain, please make sure you understand the contents of a proposal before voting.
+{% endhint %}
+
+First, go to the [governance website](https://app.realms.today/dao/PYTH) and connect your wallet.
+
+Once you connect your wallet, you can see your governance power on the top right corner. Typically your governance power will be more than your current staked tokens. This is because staked tokens get "boosted" governance power to compensate for unstaked tokens having 0 power.
+
+Example: This wallet has 30.41% of the total governance power.
```

Comment:
How about reducing the percentage so that we don't even suggest that there might be one account with that much power